src/devices/Hcsr501/README.md

```diff
@@ -0,0 +1,29 @@
+# HC-SR501
+HC-SR501 is used to detect motion based on the infrared heat in the surrounding area. 
+
+## Sensor Image
+![](sensor.jpg)
+
+## Usage
+* First, you need to create a HC-SR501 object.
+    ```C#
+    using(Hcsr501 sensor = new Hcsr501(hcsr501Pin, PinNumberingScheme.Logical))
+    {
+
```

Comment:
I'd just add a sample, put the detection logic inside and remove the comments (btw. Dispose doesn't have to be explicitly called when you use `using` - it will call it for you)